bob-common/src/mapper.rs

```diff
@@ -136,12 +136,18 @@ impl Virtual {
 
     pub fn vdisk_id_from_key(&self, key: BobKey) -> VDiskId {
         match self.distribution_func {
-            DistributionFunc::Mod => (key % self.vdisks.len() as u64)
+            DistributionFunc::Mod => (Self::get_vdisk_id_by_mod(key, self.vdisks.len()))
                 .try_into()
-                .expect("u64 to u32"),
+                .expect("usize to u32"),
         }
     }
 
+    fn get_vdisk_id_by_mod(key: BobKey, len: usize) -> usize {
+        key.iter().rev().fold([0, 1], |[rem, bmult], &byte| {
```

Comment:
`rev()` should be removed for Little Endian

Reply:
Fixed